pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Kafka-specific Bindings
+description: In this tutorial, you'll learn how how to add specific Kafka information to an AsyncAPI document using bindings.
+---
+
+## Introduction
+
+In a previous tutorial, you learnt how to manage schemas with schema registry. In this tutorial, you will learn how bindings with Kafka work by defining Kafka message key content and extending your AsyncAPI document with protocol-specific information.
+
+## Background Context
+
+Bindings are crucial to an Event-driven application as they provide protocol-specific information that abstracts the details of how protocols handle messages from the core logic of your application. Bindings improve the understanding and usability of the API by offering more setup options and context for various protocols. It includes information such as topics an application reads from or writes to, how the messages are formatted, and the rules for interacting with multiple data or messages. 
+
+In an AsyncAPI document, bindings can be appended to different document sections, such as servers, channels, or messages. They include standard information that is unique to a particular protocol. Bindings definitions allow you to specify the protocol-specific functionalities not supported by AsyncAPI's core features.
+
+You can configure several objects using Kafka bindings. However, for the scope of this tutorial, we are focusing on three levels of bindings: [server bindings](https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object), [channel bindings](https://github.com/asyncapi/bindings/tree/master/kafka#channel-binding-object) and [message bindings](https://github.com/asyncapi/bindings/tree/master/kafka#message-binding-object).
+
+
+The diagram below shows how bindings with Kafka work. 
+
+```mermaid
```

Comment:
I don't think we need such diagram that explains kafka in asyncapi

we already have a good doc and diagram in https://www.asyncapi.com/docs/concepts/asyncapi-document/adding-bindings

and your intro already links to kafka specific bindings

Reply:
Ok. This means that we don't need an image here, right?

Reply:
diagrams are always nice, but only if they bring some value in given context. As I wrote in previous comment, there is already a nice binding overview diagram, so maybe link there. Just this diagram do not help much - I basically do not understand it, there is no text that describes it - so for now I advocate to remove it

Reply:
Ok. Thank you.